lib/war.ex

```diff
@@ -12,6 +12,7 @@ defmodule War do
     {p1, p2} =
       deck
       |> Enum.map(&apply_ace_weight/1)
+      |> Enum.reverse()
```

Comment:
But I didn't understand why needs to reverse the deck before split it

Reply:
You can also reverse both files after split. In real world we always play from top card not the last so  we need to reverse

Reply:
Wow, that makes a lot of sense!